label_studio_ml/server.py

```diff
@@ -111,13 +131,48 @@ def start_server(args, subprocess_params):
     os.system('python ' + wsgi + ' ' + ' '.join(subprocess_params))
 
 
+def deploy_to_gcp(args):
+    # create project with
+    create_dir(args)
+    # prepare params for gcloud: dir with script, project id, region and service name
+    output_dir = os.path.join(args.root_dir, args.project_name)
+    project_id = os.environ.get("GCP_PROJECT")
+    if not project_id:
+        raise KeyError("Project id wasn't found in ENV variables!")
+    region = os.environ.get("GCP_REGION", "us-central1")
+    service_name = args.project_name
+    # check service name
+    # if special_match(service_name):
+    #     raise ValueError("Service name in GCP should contain only lower case ASCII letters and hyphen!")
+    # check if auth token exists
+    auth_token = subprocess.check_output(' '.join(["gcloud", "auth", "print-identity-token"]), shell=True)
+    if not auth_token:
+        raise PermissionError("You are not authentificated in gcloud! Please run gcloud auth login.")
+    # configurate project
+    subprocess.check_output(' '.join(["gcloud", "config", "set", "project", project_id]), shell=True)
+    # deploy service
+    subprocess.check_output(' '.join([
+        "gcloud", "run", "deploy",
+        service_name,
+        "--source", output_dir,
+        "--region", region,
+        "--update-env-vars", f"LABEL_STUDIO_ML_BACKEND_V2=1,LABEL_STUDIO_HOSTNAME={args.label_studio_host},LABEL_STUDIO_API_KEY={args.label_studio_api_key}"
+    ]), input=b"y", shell=True)
```

Comment:
could we add some colored message at the end, instructing us to get this links, go on Project Settings -> Machine Learning page and put this link after "Add Model"?